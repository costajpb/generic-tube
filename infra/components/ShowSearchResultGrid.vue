<template>
    <ol :class="classes.grid">
        <slot></slot>
    </ol>
</template>

<style module="classes" lang="postcss">
    .grid {
        margin-block: var(--size-4);
        list-style: none;
        padding: 0;
        display: grid;
        grid-template-columns: repeat(auto-fill, minmax(100px, 1fr));
        gap: var(--size-2);

        @media (min-width: 700px) {
            margin: 0;
        }

        > * {
            height: 150px;
        }
    }
</style>