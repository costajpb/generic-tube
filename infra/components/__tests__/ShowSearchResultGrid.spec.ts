import snapshowWrapper from "@/infra/__tests__/snapshotWrapper"
import ShowSearchResultGrid from "../ShowSearchResultGrid.vue"

describe('infra/components/ShowSearchResultGrid', () => {
    it('should render', () => {
        expect(snapshowWrapper(ShowSearchResultGrid)).toMatchSnapshot()
    })
})