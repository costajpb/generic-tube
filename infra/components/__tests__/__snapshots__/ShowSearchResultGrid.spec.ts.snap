// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`infra/components/ShowSearchResultGrid > should render 1`] = `
<ol
  class="_grid_f639ee"
>
  
  
</ol>
`;
